plugin-core/src/main/resources/messages/appland.properties

```diff
@@ -197,6 +197,8 @@ webview.findingDetails.appMapNotFound.message=The AppMap file could not be found
 
 webview.navie.title=AppMap AI: Explain
 webview.navie.chooseAppMapModule.title=Choose AppMap module
+webview.navie.webviewBroken.title=AppMap Navie
+webview.navie.webviewBroken.message=Your IDE has a known issue that typing into text input fields in the Navie is broken.\nPlease update to 2023.2 or later to fix this.
```

Comment:
Please change to:

Please update your IDE to version 2023.2 or later. Navie will not work with versions prior to 2023.2 because there is a known IDE issue that prevents typing into input fields.